src/sentry/conf/server.py

```diff
@@ -2965,9 +2965,8 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
                     "1" if settings.SENTRY_USE_ISSUE_OCCURRENCE else ""
                 ),
                 "ENABLE_AUTORUN_MIGRATION_SEARCH_ISSUES": "1",
-                "ENABLE_GROUP_ATTRIBUTES_CONSUMER": (
-                    "1" if settings.SENTRY_USE_GROUP_ATTRIBUTES else ""
-                ),
+                # TODO: remove setting
+                "ENABLE_GROUP_ATTRIBUTES_CONSUMER": "1",
```

Comment:
@wedamija maybe i should just leave the setting and default it to 1?

Reply:
Let's leave the setting for now, we can remove it in one pr once we're happy